Added documentation explaining contrib and outlining process around it (#1378)
AcademySoftwareFoundation/OpenTimelineIO: contrib/README.md

# OpenTimelineIO Contrib Area

> The contrib area was a place to host adapters and application plugins submit
> from the community, that have challenging external dependencies, or may not
> have the same level of support as the core.

Reviewer: ```suggestion
The contrib area was a place to host adapters and application plugins submitted
by the OpenTimelineIO community. Those contributions may have involved challenging external 
dependencies, and may not have the same level of support as the core.
```

The team is in the process of phasing out this area in favor of using individual
repos within the
[OpenTimelineIO GitHub Organization](https://github.com/OpenTimelineIO/).

> New contrib submissions should be created in standalone repos and can be later
> migrated to the OTIO GitHub organization after review.

Reviewer: ```suggestion
New contributions should be created in standalone repos, and those can later be
migrated to the OTIO GitHub organization after review.
```
